src/aspire/source/__init__.py

```diff
@@ -742,3 +747,4 @@ def __init__(self, im, metadata=None):
             L=im.res, n=im.n_images, dtype=im.dtype, metadata=metadata, memory=None
         )
         self._cached_im = im
+        self.unique_filters = [IdentityFilter()]
```

Comment:
I'm wondering if this would be better placed upstream (in `ImageSource`). This particular problem will crop up if we decide to subclass again for some other reason.

Reply:
Sure, moved it up to the base class.